wandb/sdk/wandb_init.py

```diff
@@ -360,14 +377,24 @@ def set_run_id(self, settings: Settings) -> None:
                 run_id=settings.run_id,
             )
 
-    def setup(
+    def compute_run_config(
```

Comment:
Perhaps "construct_..."? "Compute" has a computational vibe

Reply:
I'll change to `make_run_config`

Reply:
Did this for `compute_run_settings` as well